it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/bigtable/matchers/BigtableAsserts.java

```diff
@@ -67,4 +68,28 @@ public static List<Map<String, Object>> bigtableRowsToRecords(Iterable<Row> rows
   public static RecordsSubject assertThatBigtableRecords(Iterable<Row> rows, String family) {
     return assertThatRecords(bigtableRowsToRecords(rows, family));
   }
+
+  public static String createWritetimeSchema(Map... entries) {
+
+    ArrayList<String> schema = new ArrayList<>();
+    for (Map entry : entries) {
+      ArrayList<String> schemaRow = new ArrayList<>();
+      for (Object key : entry.keySet()) {
+        schemaRow.add(String.format("%s: %s", doubleQuote(key), doubleQuote(entry.get(key))));
+      }
+
+      String rowString = String.format("{%s}", String.join(", ", schemaRow));
+      schema.add(rowString);
+    }
+    return String.join("\n", schema);
+  }
+
+  public static String doubleQuote(Object string) {
+    return '"' + (String) string + '"';
+  }
+
+  // Convert Instant timestamp to epoch microsecond format.
+  public static long toEpochMicros(Instant time) {
+    return time.toEpochMilli() * 1000;
+  }
```

Comment:
This class is intended for truthmatchers for making resource-dependent (in this case Bigtable) asserts.

Let's move these helper methods to the IT file they are being used in, or at the very least, create a utils class in the module that these methods are intended for (`v1/src/test/bigtable/...`)

Reply:
Moved functions to CassandraToBigtableIT since that's the only place that they are used in.